frontend/src/App.tsx

```diff
@@ -2,15 +2,31 @@ import React from 'react';
 import './App.css';
 import {Navigate, Route, Routes} from "react-router-dom";
 import FoodItemGallery from "./page/FoodItemGallery";
+import UserSignUpPage from "./page/UserSignUpPage";
+import axios from "axios";
+import Cookies from "js-cookie";
+
+axios.interceptors.request.use(config => {
+    if (["put", "post", "delete"].includes(config.method || "")) {
```

Comment:
A specified axios interceptor? - Interesting!

Reply:
Elvedin showed this shortly and I think it is very elegant. So you don't have to request a new CSRF cookie before you do a normal GET. Should save time and make the App faster :)